src/index.html

```diff
@@ -10,14 +10,16 @@
   </head>
 
   <body class="body">
-    <article class="clock">
-      <div class="clock__minutes-container">
+    <div class="clock">
+
```

Comment:
no need in empty lines on start/end of block

Reply:
fix it

Reply:
fixed